modal/cloud_bucket_mount.py

```diff
@@ -133,6 +135,9 @@ def cloud_bucket_mounts_to_proto(mounts: List[Tuple[str, _CloudBucketMount]]) ->
         if mount.requester_pays and not mount.secret:
             raise ValueError("Credentials required in order to use Requester Pays.")
 
+        if not mount.key_prefix.endswith('/'):
+            raise ValueError("key_prefix will be prefixed to all object paths, so it must end in a '/'")
+
```

Comment:
Don't you have to check for `None` here?

Reply:
Are you implying we should have a separate error message for when key_prefix is empty? (since empty string does not end with a `/`)